pom.xml

```diff
@@ -77,14 +77,9 @@
             <version>1.4_rc3</version>
         </dependency>
         <dependency>
-            <groupId>io.springfox</groupId>
-            <artifactId>springfox-boot-starter</artifactId>
-            <version>3.0.0</version>
-        </dependency>
-        <dependency>
-            <groupId>io.springfox</groupId>
-            <artifactId>springfox-swagger-ui</artifactId>
-            <version>3.0.0</version>
+            <groupId>org.springdoc</groupId>
```

Comment:
Have you tested the changes: Is the Swagger UI still accessible?
Why are we using version `2.3.0`? The latest version is `2.8.8`.